src/Umbraco.Web/PropertyEditors/NestedContentPropertyEditor.cs

```diff
@@ -48,6 +49,7 @@ public NestedContentPropertyEditor(ILogger logger, Lazy<PropertyEditorCollection
             _dataTypeService = dataTypeService;
             _contentTypeService = contentTypeService;
             _localizedTextService = localizedTextService;
+            _contentTypes = new Lazy<Dictionary<string, IContentType>>(() => contentTypeService.GetAll().ToDictionary(c => c.Alias));
```

Comment:
Sorry, same issue as I mention in the block list change https://github.com/umbraco/Umbraco-CMS/pull/10235#pullrequestreview-657110265, this part will need to be reverted.

Reply:
Addressed